src/Common/src/System/Net/NetworkInformation/UnixCommandLinePing.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Text;
+
+namespace System.Net.NetworkInformation
+{
+    internal static class UnixCommandLinePing
+    {
+        // Ubuntu has ping under /bin, OSX under /sbin.
+        private static readonly string[] s_binFolders = { "/bin", "/sbin" };
```

Comment:
I'd added slashes at the end of these paths.  It'd be good to do so here as well.  Otherwise, Path.Combine is forced to add them, a slightly more expensive code path.